my_enumerable.rb

```diff
@@ -0,0 +1,23 @@
+module MyEnumerable
+  def all?
+    @list.each do |e|
+      return false unless yield e
+    end
+    true
+  end
+
+  def any?
+    @list.each do |e|
+      return true if yield e
+    end
+    false
+  end
+
+  def filter
+    filtered = []
+    @list.each do |e|
+      filtered.push(e) if yield e
+    end
```

Comment:
- [ ] Using the `each` method given by ruby is not allowed kindly implement your own `each` method in `MyList` class 😄